src/Evolution/Systems/CurvedScalarWave/Worldtube/Tags.hpp

```diff
@@ -0,0 +1,14 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "Options/Options.hpp"
+
+namespace CurvedScalarWave::Worldtube {
```

Comment:
Add dox to this namespace please. For more extensive docs you can add a `Worldtube.hpp` file that only documents the namespaces.